src/web/components/Branding.tsx

```diff
@@ -44,22 +46,14 @@ export const Branding: React.FC<{
         <div className={brandingStyle}>
             <div className={brandingLabelStyle}>{branding.logo.label}</div>
             <div className={brandingLogoStyle}>
-                <MaintainAspectRatio
-                    height={branding.logo.dimensions.height}
-                    width={branding.logo.dimensions.width}
+                <a
+                    href={branding.logo.link}
+                    data-sponsor={branding.sponsorName.toLowerCase()}
+                    rel="nofollow"
+                    aria-label={`Visit the ${branding.sponsorName} website`}
                 >
-                    <a
-                        href={branding.logo.link}
-                        data-sponsor={branding.sponsorName.toLowerCase()}
-                        rel="nofollow"
-                        aria-label={`Visit the ${branding.sponsorName} website`}
-                    >
-                        <img
-                            src={branding.logo.src}
-                            alt={branding.sponsorName}
-                        />
-                    </a>
-                </MaintainAspectRatio>
+                    <img src={branding.logo.src} alt={branding.sponsorName} />
```

Comment:
I think we want height and width here again?

Reply:
thanks, will add those

Reply:
Turns out that we lose the fixed aspect ratio by adding these, I'll have to remove this in another PR I think.